ledger/participant-integration-api/src/main/scala/platform/store/backend/common/ParameterStorageBackendTemplate.scala

```diff
@@ -78,20 +80,18 @@ private[backend] object ParameterStorageBackendTemplate extends ParameterStorage
       ParameterStorageBackend.IdentityParams(ledgerId, participantId)
     }
 
-  private val LedgerEndParser: RowParser[Option[ParameterStorageBackend.LedgerEnd]] =
+  private val LedgerEndParser: RowParser[ParameterStorageBackend.LedgerEnd] =
     LedgerEndOffsetParser ~ LedgerEndSequentialIdParser ~ LedgerEndStringInterningIdParser map {
       case Some(lastOffset) ~ Some(lastEventSequentialId) ~ Some(lastStringInterningId) =>
-        Some(
-          ParameterStorageBackend.LedgerEnd(
-            lastOffset,
-            lastEventSequentialId,
-            lastStringInterningId,
-          )
+        ParameterStorageBackend.LedgerEnd(
+          lastOffset,
+          lastEventSequentialId,
+          lastStringInterningId,
         )
       case _ =>
```

Comment:
is there really a case where ledger end is read before initialization? (if not I would recommend making this simple, and even failing with exception if done otherwise)